### DISCUSSION TITLE
[2차회의관련] 회의 느낀점 및 보완점 작성 관련 (24.11.20)

### DISCUSSION THREAD

**Discussion by apricity2u** [codeCommonsLab/dev_doc_web#18] [General]
금일 회의에서는 각자 작성한 프로젝트 개요를 공유하는 시간을 가졌습니다.  (Notion, Figma, Obsidian, etc..)
오늘은 급하게 회의를 주관하게 되어 진행에 미숙한 부분이 많았는데, 너그러운 양해 부탁드립니다^^

따라서 각자 느낀점 / 보완할 점 / 회의 아이디어 등을 작성해주시면, 금일 피드백을 바탕으로 차후 회의를 더 원활하게 진행하고자 합니다. 편안하게 의견 남겨주세요!

**Comment by ssdeok** [comment]
과정들을 글로 적어보는게 아직은 힘들어서 간단하게만 해보았는데, 다른 분들이 하신것들을 보고 많이 배워갑니다. 유익한 시간이었습니다. 다들 고생하셨습니다.

**Comment by kska01** [comment]
다양한 생각들과 방법들이 공유되어서 좋았습니다. 갑작스레 시작했지만 좋은 출발이었던 것 같습니다.

다음 주 수요일 모임에서 이야기 나누었으면 좋을 주제를 적어봅니다.
- 일단 다음 주 주제는 무엇으로 할지 정해야 할 것 같습니다.
- 개발 흐름 정리?, 생각정리? 이 주제로 하는 스터디에 참여 유무 정해서 인원 확정하기
- 스터디에 참여할 사람들은 스터디가 어떤 방법으로 운영이 되었으면 좋을지 생각해 보고 공유하기
  - 회의실 예약자 순서 정하기
  - 필요한 자료나 주제 등의 조사 및 확정 등을 어떻게 할지... 등등

일단 지금은 이 정도 생각이 나네요.. 생각이 더 나면 좀 더 추가하겠습니다. 

갑작스럽게 부탁한 리더 역할을 맡아 주신 하은님 고맙습니다. 고생하셨어요!! 

어제 다들 고생하셨습니다!! 모두 화이팅입니다!!!
